extension/extension.go

```diff
@@ -65,6 +65,14 @@ func (f *factory) Stability() component.StabilityLevel {
 	return f.extensionStability
 }
 
+func (f *factory) Create(ctx context.Context, set Settings, cfg component.Config) (Extension, error) {
```

Comment:
I was thinking this morning about exactly same thing. I would deprecate func at line 35 since no longer useful and don't want to maintain it.

Reply:
Done